api/0.1.txt

```diff
@@ -671,3 +671,13 @@ package androidx.view {
 
 }
 
+package androidx.widget {
```

Comment:
You should not be changing this file I believe.

Reply:
Ops! I saw a PR yesterday modifying it, but it also reverted later. Reverted.